ci/matlab.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This is a basic workflow to help you get started with MATLAB actions
+# For more information see: https://github.com/matlab-actions/overview
+
+name: MATLAB
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Set up MATLAB on the GitHub hosted runner
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v1
+
+      - name: Run tests and generate artifacts
+        uses: matlab-actions/run-tests@v1
+        with:
+          source-folder: CODE
+          code-coverage-cobertura: code-coverage/coverage.xml
+          test-results-junit: test-results/results.xml
+
+      - name: Upload code coverage report
+        uses: actions/upload-artifact@v2
+        with:
+          name: code-coverage
+          path: code-coverage/coverage.xml
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: test-results
+          path: test-results/results.xml
+
+      # Uncomment this step to execute a MATLAB script, function, or statement
+      #- name: Run script
+      #  uses: matlab-actions/run-command@v1
+      #  with:
+      #    command: SCRIPT
+
+      # Uncomment this step to run a MATLAB build
```

Comment:
(Unresolved comment from previous PR)

@davidbuzinski : Might be good to link to buildtool docs or run-build docs. I'm not sure if users will know what buildplan means in this context.

Reply:
Updated the comment a bit. Will the [doc link](https://github.com/matlab-actions/overview) at line 5 of this template be enough?

Reply:
As above, instead of the "Uncomment this step" can we say something like "As an alternative to run-tests, you can use run-build to execute a MATLAB build."